CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fix folio in `corn`
 * Style `equations` in `contemporary-math`
 * Clean up `common` files
+* Added `img.scaled-down` styling to `FigureImage` shapes & components in `webview`
```

Comment:
looks like this is a duplicate line (probably left over from git adventures). could you remove this line, and add a line to the changelog describing this change, and make sure it goes under the `Unreleased` section above?